### DISCUSSION TITLE
Adjusting subtitle line duration from end time

### DISCUSSION THREAD

**Discussion by dgoryeo** [SubtitleEdit/subtitleedit#6770] [Ideas]
Hi,

Would it be possible to adjust the duration of subtitle lines by keeping the end time fixed, and adjusting the start time to match the duration: Example:
```

2
00:00:02,800 --> 00:00:014,040   <==duration adjustment needed
This is a sub line example            

2
00:00:10,800 --> 00:00:14,040      <==duration adjusted
This is a sub line example            

```

This wil specially be helpful for adjusting subs from Whisper --much of the mistimings can be fixed this way in my experience. At the moment I do this manually which is quite painstaking :)  If anyone can think of a good script to automate it I'd be grateful :)

**Comment by davexnet** [comment]
I think this is a good idea, a way to lock the end time.
Particularly since SE works with whisper directly, and this kind of issue comes up often
